src/components/editor/markdown-renderer/markdown-it-plugins/plantuml-error.ts

```diff
@@ -0,0 +1,20 @@
+import MarkdownIt, { Options } from 'markdown-it/lib'
+import Renderer, { RenderRule } from 'markdown-it/lib/renderer'
+import Token from 'markdown-it/lib/token'
+
+export const plantumlError: MarkdownIt.PluginSimple = (md) => {
+  const defaultRenderer: RenderRule = md.renderer.rules.fence || (() => '')
+  md.renderer.rules.fence = (tokens: Token[], idx: number, options: Options, env, slf: Renderer) => {
+    const token = tokens[idx]
+    const defaultRender = defaultRenderer(tokens, idx, options, env, slf)
+    if (token.info === 'plantuml') {
+      return `
+        <p class="alert alert-danger">
+          PlantUML plugin is enabled but not properly configured.
+        </p>
+        ${defaultRender}
```

Comment:
Why are you rendering the source code? if plantuml is enabled I wouldn't expect a code block.